components/text/Preformatted.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Preformatted = ({ className, ...rest }) => {
+    return <pre style={{ overflow: 'auto' }} className={`bg-global-muted p0-5 m0 ${className}`} {...rest} />;
```

Comment:
Ask @nico3333fr for a class to use `overflow: 'auto'` please.

Reply:
You should have at least `mb1` on this component since it has a block structure.

Reply:
You may use `scroll-if-needed` to have `overflow: auto` ;)